app/src/main/java/org/flyve/mdm/agent/EditUserActivity.java

```diff
@@ -17,10 +24,15 @@
 
 import org.flyve.mdm.agent.core.user.UserController;
```

Comment:
You will re-use these resources for the Dashboard.

I think will better to use `org.flyve.core.user.UserController` instead `org.flyve.mdm.agent.core.user.UserController`